docs/spfx/sharepoint-subscription-edition-component.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Upgrade components in SharePoint Framework development with SharePoint Server Subscription Edition
+description: SharePoint Server Subscription Edition (SE) supports to ungrade individual components within SharePoint Framework when developing client-side web parts.
+ms.date: 09/08/2023
+ms.localizationpriority: high
+---
+
+
+# Upgrade components in SharePoint Framework development with SharePoint Server Subscription Edition
+
+SharePoint Server Subscription Edition supports SharePoint Framework client-side web parts in classic and modern pages, and extensions in modern pages.
+
+When developing a client-side web part, you can only use the specific versions of components supported by the SharePoint Framework to make the web part compatible with your SharePoint Server.
+
+To use more modern versions of the SharePoint Framework (SPFx) components, this article introduces some ways to upgrade individual components without upgrading the major version of SPFx.
+
+> [!NOTE]
+> Because the component-wide upgrades were not made as part of a broader update to SPFx, some manual configuration of the development environment might be completed in order to utilize them.
+
+> [!IMPORTANT]
+> Only SharePoint Server Subscription Edition (SE) supports the component-wide upgrades for the SPFx development since the [Version 23H2](/sharepoint/what-s-new/new-and-improved-features-in-sharepoint-server-subscription-edition-23h2-release#sharepoint-framework-spfx-component-upgrades) feature update. SharePoint Server 2019 and older SharePoint Server products do not support that.
+
+## Upgrade React and Fluent UI React
+
+The SharePoint Framework v1.4.1 natively supports React v15.8 and Fluent UI React ([formerly Office UI Fabric React](https://devblogs.microsoft.com/microsoft365dev/ui-fabric-is-evolving-into-fluent-ui/)) v5.17.
+
+If you want to use React v16 or Fluent UI React v7 in your custom web parts developed for SharePoint Server with the SharePoint Framework v1.5.0, here are the instructions:
+
+1. Modify the "dependencies" and "devDependencies" in the npm project:
+
+    Use Yeoman to generate scaffold code, and modify the dependences in the **package.json**:
+
+    - Change the **react** and **react-dom** versions to "16.13.1", the **@types/react** version to "16.9.51", and the **@types/react-dom** version to "16.9.8".
+
+    - Add **@microsoft/office-ui-fabric-react-bundle**: "1.15.0" in the "dependencies" section.
+
+    - Change the **@microsoft/sp-build-web** and **@microsoft/sp-module-interfaces** versions to "1.9.1" in the "devDependencies" section.
+
+    - Add **gulp-json-editor**: "2.2.1", **@microsoft/sp-tslint-rules**: "1.9.1", and **@microsoft/rush-stack-compiler-2.9**: "0.7.16" to the "devDependencies" section.
+
+    - Change the rest of the SPFx components from version "1.4.0" or "1.4.1" to "1.5.0".

```

Comment:
What about those who deployed SPFx v1.5.1 as part of the 23H1-supported version?